README.md

```diff
@@ -0,0 +1,71 @@
+# Preface
+
+This assume you have installed codius, if you havent please do so first. If you are using the community installer, this install guide will be added soon on that script.
```

Comment:
I wouldn't make any promises regarding the Community Installer. We don't have a direct relationship with maintenance of the community installer, so I think it's best not to imply that we have that kind say in its development cycle, and vice versa.